sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,247 @@
+## APIs:
+class ServiceBusClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        credential : TokenCredential,
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = None,
+    ):
+
+    def __enter__(self):
+    def __exit__(self, *kwargs):
+    def close(self):
+
+    @classmethod
+    def from_connection_string(cls, conn_str, **kwargs):
+
+    def get_queue_sender(self, queue_name, **kwargs) -> ServiceBusSender:
+    def get_queue_receiver(self, queue_name, **kwargs) -> ServiceBusReceiver:
+    def get_topic_sender(self, topic_name, **kwargs) -> ServiceBusSender:
+    def get_subscription_receiver(self, topic_name, subscription_name, **kwargs) -> ServiceBusReceiver:
+
+class ServiceBusSender:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        credential: TokenCredential,
+        queue_name: str = None,
+        topic_name: str = None,
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = None,
+        retry_total : int = 3,
+        retry_backoff_factor : float = 30,
+        retry_backoff_maximum : int = 120  # cur_retry_backoff = retry_backoff_factor * (2^cur_retry_time)
+    ) -> None:
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str : str,
+        queue_name: str = None,
+        topic_name: str = None,
+        **kwargs
+    ) -> ServiceBusSender:
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, Any]:
+
+    def create_batch(
```

Comment:
I'm gonna regret saying this, because I recall _the exact_ same discussion in EH, but if the typical flow is:
create_batch -> send
wouldn't it be more sane to just send_batch?  

I could see this being disadvantageous especially if we eventually have a send that could take a list of messages or batches, but otherwise I'm having trouble justifying the intermediary step.